nilearn/decomposition/multi_pca.py

```diff
@@ -174,21 +105,15 @@ def __init__(self, n_components=20, smoothing_fwhm=None, mask=None,
                  t_r=None, memory=Memory(cachedir=None), memory_level=0,
                  n_jobs=1, verbose=0,
                  ):
-        self.mask = mask
-        self.memory = memory
-        self.memory_level = memory_level
-        self.n_jobs = n_jobs
-        self.verbose = verbose
-        self.low_pass = low_pass
-        self.high_pass = high_pass
-        self.t_r = t_r
-
-        self.do_cca = do_cca
-        self.n_components = n_components
-        self.smoothing_fwhm = smoothing_fwhm
-        self.target_affine = target_affine
-        self.target_shape = target_shape
-        self.standardize = standardize
+
+        SinglePCA.__init__(self, n_components=n_components,
+                           smoothing_fwhm=smoothing_fwhm,
+                           mask=mask,
+                           standardize=standardize, target_affine=target_affine,
+                           target_shape=target_shape,
+                           low_pass=low_pass, high_pass=high_pass,
+                           t_r=t_r, memory=memory, memory_level=memory_level,
+                           n_jobs=n_jobs, verbose=verbose, do_cca=do_cca)
```

Comment:
OK, I see that it would make the code much more complex. So let's leave it there. It just needs to be well documented there.